evil-collection.el

```diff
@@ -32,6 +32,16 @@
 ;; This is so because many users find it confusing.
 
 ;;; Code:
+
+;; FIXME: Is this the best way to do this?
+(when (or
```

Comment:
I think this can simply check for `(featurep 'evil-integration)` after the `(require 'evil)`.

Reply:
featurep check is OK with me though it does have a little bit of indirection. :)